test/example/c-api-kitchen-sink.c

```diff
@@ -765,6 +765,12 @@ void test_binaries() {
   assert(BinaryenModuleValidate(module));
   printf("module loaded from binary form:\n");
   BinaryenModulePrint(module);
+
+  // write the s-expr representation of the module.
+  BinaryenModuleWriteSExpr(module, buffer, 1024);
```

Comment:
please test the other new API call as well

Reply:
Done/